### ISSUE TITLE
FieldArray gives my rendered component a prop called `value`, is this a public API?

### ISSUE THREAD

**Issue by GavinThompson** [final-form/react-final-form-arrays#13]
### Are you submitting a **bug report** or a **feature request**?

Bug report, maybe? Docs?

### What is the current behavior?

The component I pass to `<FieldArray>` in the `component` prop receives a prop called `value`.  Is this a public/stable API?

I don't see this in `FieldArrayRenderProps`, either in [the docs](https://github.com/final-form/react-final-form-arrays#fieldarrayrenderprops) or [the flow types](https://github.com/final-form/react-final-form-arrays/blob/master/src/types.js.flow#L5).

### Sandbox Link

Here's the FieldArray example, adapted to show I can read from the `value` prop.

https://codesandbox.io/s/n1nn8rpjwl

### What's your environment?

Same as the example codesandbox.

### Other information

Ultimately what I'm looking for is a way to read from the FieldArray programmatically/imperatively.

In redux-form I could use `.get()` and/or `.getAll()`, but this isn't available in `react-final-form`. 

Any guidance?

**Comment by erikras** [comment]
Wow. Brilliant observation! I totally overlooked that.

Let me think about it, but without thinking too much about it, I'm leaning towards the redux-form API...

**Comment by GavinThompson** [comment]
Excellent, thanks for the quick response! Will hold off on any more changes on our end and circle back in the near future. Thanks again! 😄

**Comment by shrynx** [comment]
@erikras It seems this been removed from the latest version, any plans for alternate api ?

**Comment by davidroeca** [comment]
It would be really nice to have the `get` functionality. As a work-around, I had success with the `Field` component; it works with both the entire FieldArray name as well as the field name from a call to `fields.map()`:

```jsx
<FieldArray name='myArray'>
  {({ fields }) => (
    <React.Fragment>
      {fields.map((name) => (
        <Field name={name} subscription={{value: true}}>
          {({ input: { value: field }}) => (
            <div>{JSON.stringify(field, null, 2)}</div>
          )}
        </Field>
      )}
      <Field name='myArray' subscription={{value: true}}>
        {({ input: { value: fieldValues }}) => (
          <div>{JSON.stringify(fieldValues, null, 2)}</div>
        )}
      </Field>
    </React.Fragment>
  )}
</FieldArray>
```

**Comment by erikras** [comment]
Published fix in [`v3.1.1`](https://github.com/final-form/react-final-form-arrays/releases/tag/v3.1.1).
